update_tdr_realm.py

```diff
@@ -0,0 +1,85 @@
+import requests
+import json
+import sys
+import os
+
+from update_realm_configuration import update_realm_configuration
+
+stage_to_url = dict(
```

Comment:
This might be too hard to be feasible, but is it possible to get these values from AWS so they don't have to be hard-coded into this script?

Reply:
Could potentially store them as SSM parameters, and get the Jenkins job to retrieve the value, as with the TDR account numbers. Then pass it in as param to the python script.